### PR TITLE
Update ServiceBus Readme to include an example

### PR DESCRIPTION
**Pull request by devigned**

### CODE REVIEW

File: lib/services/serviceBus/README.md [hunk 1]
```diff
@@ -14,9 +14,66 @@ npm install azure-sb
 
 ## How to Use
 
-### Authentication
+```node
+var azure = require('azure');
 
-### Create the ServiceBus service
+function checkForMessages(sbService, queueName, callback) {
+    sbService.receiveQueueMessage(queueName, { isPeekLock: true }, function (err, lockedMessage) {
```
Comment (yugangw-msft) [inline lib/services/serviceBus/README.md]: please make indent at 2

File: lib/services/serviceBus/README.md [hunk 2]
```diff
@@ -14,9 +14,66 @@ npm install azure-sb
 
 ## How to Use
 
-### Authentication
+```node
+var azure = require('azure');
```
Comment (yugangw-msft) [inline lib/services/serviceBus/README.md]: it is unclear this code has nothing to do with the Authentication, should we remove it?

File: lib/services/serviceBus/README.md [hunk 3]
```diff
@@ -14,9 +14,66 @@ npm install azure-sb
...
+       if (err) {
+           console.log('Failed Tx: ', err);
+       } else {
+           console.log('Sent ' + msg);
+       }
+    });
+}
+
+var connStr = process.argv[2] || process.env.CONNECTION_STRING;
+if (!connStr) throw new Exception('Must provide connection string');
```
Comment (yugangw-msft) [inline lib/services/serviceBus/README.md]: Use "Error"?

File: lib/services/serviceBus/README.md (hunk 2 above)
Comment (devigned) [inline lib/services/serviceBus/README.md]: the authentication line is deleted already

Comment (yugangw-msft) [inline lib/services/serviceBus/README.md]: ah, didn't read it carefully.

**Pull request merged**